onnxmltools/utils/metadata_props.py

```diff
@@ -0,0 +1,57 @@
+import warnings
+from ..convert.common.case_insensitive_dict import CaseInsensitiveDict
+from ..proto import onnx, onnx_proto
+from distutils.version import StrictVersion
+
+
+KNOWN_METADATA_PROPS = CaseInsensitiveDict({
+    'Image.BitmapPixelFormat': ['gray8', 'rgb8', 'bgr8', 'rgba8', 'bgra8'],
+    'Image.ColorSpaceGamma': ['linear', 'srgb'],
+    'Image.NominalPixelRange': ['nominalrange_0_255', 'normalized_0_1', 'normalized_1_1', 'nominalrange_16_235'],
+})
+
+
+def _validate_metadata(metadata_props):
+    '''
+    Validate metadata properties and possibly show warnings or throw exceptions.
+
+    :param metadata_props: A dictionary of metadata properties, with property names and values
```

Comment:
What are typical (key, value) pairs? Could you please add two examples into the doc string? Thanks.